plugins/modules/win_domain_computer.py

```diff
@@ -47,6 +49,7 @@
     description:
       - Specifies the X.500 path of the Organizational Unit (OU) or container
         where the new object is created. Required when I(state=present).
+      - Special characters must be escaped, see U(https://docs.microsoft.com/en-us/previous-versions/windows/desktop/ldap/distinguished-names)
```

Comment:
Is this just a limitation of the module or special characters in OU's in general?

Also you can use `L(Link name,URL)` to shorten the URL as it appears in the docs

Reply:
This is a general thing you'd have to do when passing OU names to modules/libraries. It's not really specific to this one, but thought it was worth pointing out.

I thought the `L()` syntax only worked on links within docs? I had always wanted to use it for outside links but thought it was impossible:

https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#linking-within-module-documentation
> For modules in a collection, you can only use L() and M() for content within that collection. Use U() to refer to content in a different collection.

Reply:
Links within the docs are meant to use the new-ish R() syntax https://github.com/ansible/ansible/pull/69832. I'll double check but I've used `L()` before as an equivalent for `[label](link)`.